docs/whats_new/version_ongoing.rst

```diff
@@ -43,3 +43,7 @@ adapt the record auxiliary to fit with the brand new double threaded auxiliary i
 Acroname Auxiliary adaption
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 adapt the acroname auxiliary to fit with the brand new double threaded auxiliary interface
+
+Agnostic CCSocketCan
```

Comment:
The  "whats new " should actually only contain new features and not fixes. So the most interesting part for users.

You should squash everything down to one single commit message, so the changes will then appear in the Changelog.
fix: ...